### DISCUSSION TITLE
feat: We can search nearest hospital on Google Maps with a button on drawer menu

### DISCUSSION THREAD

**Discussion by khnbzkrt** [acikkaynak/deprem-yardim-frontend#723] [Ideas]
## Define your request
We can use google maps to find nearby coordinates.

** discord username: Hakan Angın#9836  **

## Other Evaluated Alternatives
I think we can display the closest hospital information to the area we are driving with on google maps with the following url pattern.
https://www.google.com/maps/search/hastane/@37.5500685,35.1300749,10z

## Extra context
_Not have_

**Comment by 0xCakin** [comment]
Instead perhaps we can load the hospital data as a layer. 
When user clicks one of the locations, the app suggests the closest hospital.

**Comment by 0xCakin** [comment]
Let's move this to ideas. I will mark as p0 so we can quickly circle.
